packages/graphql/src/classes/Neo4jGraphQL.ts

```diff
@@ -110,7 +111,12 @@ class Neo4jGraphQL {
         }
     }
 
-    public async getSchema(): Promise<GraphQLSchema> {
+    public async getSchema(
+        { experimental }: { experimental: boolean } = { experimental: false }
+    ): Promise<GraphQLSchema> {
```

Comment:
Interesting that you have done this here, I was expecting this to be in the constructor - for instance, this won't work with a subgraph schema. I think we should fully consider where this goes!

Reply:
moved it to constructor